src/cwe_checker_lib/src/abstract_domain/data.rs

```diff
@@ -234,6 +234,19 @@ mod tests {
                 contains_top_values: false,
             }
         }
+
+        pub fn mock_from_absolute_value(absolute_value: T) -> Self {
```

Comment:
`DataDomain` already implements the traits `From<Bitvector>` and `From<T>` which do exactly the same.